### ISSUE TITLE
How do I decode a shc:/... string?

### ISSUE THREAD

**Issue by repzilon** [angusmillar/SmartHealthCard#5]
I would like to try out decoding, but I don't understandand where should I put the string starting with "shc:/" I get after reading the QR code. Can you explain me please?

**Comment by angusmillar** [comment]
Hi @repzilon, I do apologise as this feature is not incorporated into the library as yet. I will be honest, it completely slipped my mind as I personally don't decode QR Codes. But it certainly needs to include this as it completes the loop. I will work on it in the coming weeks.

**Comment by angusmillar** [comment]
Hi @repzilon, You had me confused and thinking I did not implement this but it turns out I did. It's just not demonstrated within the ReadMe file. I will update the readme file a bit later to explain to others. Yet for you now the quick answer can be seen in the test case found here: [SmartHealthCardQRCodeEncoderTest.cs](https://github.com/angusmillar/SmartHealthCard/blob/master/SmartHealthCard.Test/SmartHealthCardQRCodeEncoderTest.cs). Look at line 144 where I feed in a List<string> named 'QRCodeRawDataList'. This is a string list where each string is a QR Code's raw data (e.g. "shc:/"). Remember sometimes for large payloads there might be many QR Codes that together represent a single JWS token, thus why it is a list. If you only have a single QR Code then it's just a list containing one entry.

That list then goes into the SmartHealthCardQRCodeDecoder.GetToken() and then that returns a JWS token which you then can further decode and verity.  

Have a look at that test case I linked to and it should be fairly self-explanatory.

I will try and update the ReadMe file to also show this for others.

Thanks for pointing it out.
